From 3755965f82a7262eb0b1cc9f67c2761995d11aa9 Mon Sep 17 00:00:00 2001
From: Pradeep Sriram <pradeep.sriram@amlogic.com>
Date: Mon, 17 May 2021 21:00:19 -0700
Subject: [PATCH 2/3] arm64: cpuinfo: Add "model name" in /proc/cpuinfo

PD#SWPL-48418

Problem:
cpuinfo does not show model name
in 32 bit user space

Solution: remove compat check

Verify: ah212

Signed-off-by: Pradeep Sriram <pradeep.sriram@amlogic.com>
Change-Id: I6bef774816e756f4f298d4c038752ac86518f993
---
 arch/arm64/kernel/cpuinfo.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/arch/arm64/kernel/cpuinfo.c b/arch/arm64/kernel/cpuinfo.c
index be9d39d56c07..d07c930ebe48 100644
--- a/arch/arm64/kernel/cpuinfo.c
+++ b/arch/arm64/kernel/cpuinfo.c
@@ -145,8 +145,7 @@ static int c_show(struct seq_file *m, void *v)
 		 * "processor".  Give glibc what it expects.
 		 */
 		seq_printf(m, "processor\t: %d\n", i);
-		if (compat)
-			seq_printf(m, "model name\t: ARMv8 Processor rev %d (%s)\n",
+		seq_printf(m, "model name\t: ARMv8 Processor rev %d (%s)\n",
 				   MIDR_REVISION(midr), COMPAT_ELF_PLATFORM);
 
 		seq_printf(m, "BogoMIPS\t: %lu.%02lu\n",
-- 
2.25.1

